package.json

```diff
@@ -27,44 +27,45 @@
   ],
   "dependencies": {
     "concat-stream": "^2.0.0",
-    "deepcopy": "^2.0.0",
+    "deepcopy": "^2.1.0",
     "dicer": "^0.3.0",
-    "json-text-sequence": "^0.1.1",
+    "gulp-update": "0.0.2",
```

Comment:
Does the Node.js API have a direct dependency on gulp-update?  Or only an indirect dependency via gulp.

If the Node.js API doesn't make direct use of gulp-update, it's better not to list the dependency because requiring the correct version of gulp will also pick up the correct version of gulp-update.

Reply:
Done.